CLI.md

```diff
@@ -293,4 +300,4 @@ For `model_path` in nexa commands, it's better to follow the standard format to
 
 - `gemma-2b:q4_0`
 - `Meta-Llama-3-8B-Instruct:onnx-cpu-int8`
-- `alanzhuly/Qwen2-1B-Instruct:q4_0`
\ No newline at end of file
+- `alanzhuly/Qwen2-1B-Instruct:q4_0`
```

Comment:
not able to find this on modelhub
![image](https://github.com/user-attachments/assets/5b94802e-01df-464e-a880-2d13f0a9c7bc)
also, please use company's repo. rather than personal ones